### PR TITLE
feature delete tweet method (#18)

### LINKED ISSUE #18: feature - method to delete tweet
Complete delete tweet functionality.

### PR DESCRIPTION
**Pull request by cags84**
- Closes #18 - The method to delete tweet is implemented.
- Updated - the destroy method in users was updated.
- First attempt at code documentation

### CODE REVIEW

File: api/middleware/authorization.js [hunk 1]
```diff
@@ -19,8 +20,26 @@ const usersAuthorization = async (req, res, next) => {
   }
 };
 
-const tweetsAuthorization = (req, res, next) => {
-  next();
+/**
```
Comment (jestrade) [inline api/middleware/authorization.js:23]: excelente

File: api/services/tweetService.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const Tweet = require("../tweets/model");
+
+const verifyIfUserIsOwnOfTweet = async (userId, tweetId) => {
```
Comment (jestrade) [inline api/services/tweetService.js]: if owns tweet

File: api/services/tweetService.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  const validateUser = await Tweet.find({
```
Comment (jestrade) [inline api/services/tweetService.js:4]: excelente

**Review by jestrade — APPROVED** [review]
LGTM

File: api/services/tweetService.js (hunk 2 above)
Comment (cags84) [reply api/services/tweetService.js]: change successful

**Comment by cags84** [comment]
changes applied

**Pull request merged**